src/modules/org-data/index.tsx

```diff
@@ -14,3 +21,21 @@ function getCurrentOrg(): WEBSHOP_ORGS {
 
   throw new Error('NEXT_PUBLIC_PLANNER_ORG_ID required but missing');
 }
+
+function getMapboxData(): MAPBOX_DATA {
+  const accessToken = process.env.NEXT_PUBLIC_MAPBOX_API_TOKEN;
+  const style = process.env.NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL;
+  const defaultLat = process.env.NEXT_PUBLIC_MAPBOX_DEFAULT_LAT;
+  const defaultLng = process.env.NEXT_PUBLIC_MAPBOX_DEFAULT_LNG;
+
+  if (!accessToken || !style || !defaultLat || !defaultLng) {
+    throw new Error('Mapbox data required but missing');
+  }
+
+  return {
+    accessToken,
+    style,
+    defaultLat: Number(defaultLat),
+    defaultLng: Number(defaultLng),
```

Comment:
We have kind of control of this as it is part of env setup, but I think still this can be more proper parsing:
```suggestion
    defaultLat: parseFloat(defaultLat, 10),
    defaultLng: parseFloat(defaultLng, 10),
```